docs/topics/api/addons.rst

```diff
@@ -47,6 +47,7 @@ This endpoint allows you to search through public add-ons.
     :query int page: 1-based page number. Defaults to 1.
     :query int page_size: Maximum number of results to return for the requested page. Defaults to 25.
     :query string platform: Filter by :ref:`add-on platform <addon-detail-platform>` availability.
+    :query string tag: Filter by exact tag name. Multiple tag names can be specified, separated by comma(s).
```

Comment:
I quickly looked it up and it seems that the tag text theoretically can contain anything since it's just a regular `CharField` and I don't see any normalization going on anywhere. I'm guessing though tags are admin-defined right? Just making sure we don't fail because someone suddenly decides to put a comma in a tag.